### PR TITLE
feat: add follow/unfollow & fix bookmark issue

### PR DESCRIPTION
**Pull request by anik31**
### [Live Link](https://vacay-book-git-profile-anik31.vercel.app/)

### CODE REVIEW

File: src/features/profile/userSlice.js [hunk 1]
```diff
@@ -42,7 +82,70 @@ const userSlice = createSlice({
...
+    },
+    [editUserInfo.rejected]: (state, { payload }) => {
+      state.isLoading = false;
+      state.error = payload;
+      toast({
+        title: `${state.error}`,
+        status: "error",
+        position: "top-right",
+        isClosable: true,
+        duration: 3000
```
Comment (Maianki) [inline src/features/profile/userSlice.js:110]: Here you can add common props directly to the toast container.

**Review by Maianki — APPROVED** [review]
Great job. Neat and clean code. Just added one suggestion.

File: src/features/profile/userSlice.js (hunk 1 above)
Comment (anik31) [reply src/features/profile/userSlice.js:110]: Will do it

File: src/features/profile/Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { DisplayProfile } from "./components/DisplayProfile";
+import { useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+import { DisplayPost } from "features";
+import { Center, Spinner } from '@chakra-ui/react';
+
+export function Profile(){
+    const {allUsers} = useSelector(store=>store.user);
+    const {posts, isLoading} = useSelector(store=>store.post);
+    const params = useParams();
```
Comment (sainath7878) [inline src/features/profile/Profile.jsx:10]: suggestion: You can directly destructure params here and use them wherever required.

Comment (anik31) [reply src/features/profile/Profile.jsx:10]: Thanks for the suggestion

**Review by sainath7878 — APPROVED** [review]
Code is very neat and understandable. Awesome work on the UI. Check the suggested change and you're good to merge.

**Pull request merged**